styles.scss

```diff
@@ -290,6 +298,7 @@ a:hover {
   border: solid black 3px;
   width: 50rem;
   margin: auto;
+  border-radius: 5px;
   //animation
   position: relative;
   animation-name: mymove;
```

Comment:
give a more descriptive name to your animation, for example, what's the animation doing?  - floating - spinning - rotating, etc.